src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -48,7 +49,7 @@ public App()
 		/// will be used such as when the application is launched to open a specific file.
 		/// </summary>
 		/// <param name="e">Details about the launch request and process.</param>
-		protected override void OnLaunched(LaunchActivatedEventArgs e)
+		protected override async void OnLaunched(LaunchActivatedEventArgs e)
```

Comment:
You should launch an "async void" instead of converting the `OnLaunched` to async.

... anyway it looks like you have no `await` in your method.

Reply:
fixed :-)